app/Components/Sudoku Board/SudokuBoard.tsx

```diff
@@ -449,18 +468,7 @@ const SudokuBoard = (props: SudokuBoardProps) => {
       inputValue == "E" // e and E are for erase
     ) {
       eraseSelected();
-    } else if (inputValue == "u" || inputValue == "U") {
-      undo();
-    } else if (inputValue == "p" || inputValue == "P") {
-      handlePause(sudokuBoard, navigation);
-    } else if (
-      inputValue == "t" ||
-      inputValue == "T" ||
-      inputValue == "n" ||
-      inputValue == "N"
-    ) {
-      toggleNoteMode();
-    } else if (sudokuBoard.selectedCell) {
+    } else {
```

Comment:
nit: this else isn't needed, can just add the code below and not have it nested another layer

Reply:
I've updated as you suggested, good callout